### PR TITLE
Implement the cva6 configurations defined in cva6 specification

### PR DESCRIPTION
**Pull request by JeanRochCoulon**
Implement the cva6 configurations defined in cva6 specification
target variable can take cv64a6_imacfd_sv39, cv32a6_imac_sv0,
                                       cv32a6_imac_sv32, cv32a6_imacf_sv32

### CODE REVIEW

File: cva6/sim/cva6.yaml
Comment (MikeOpenHWGroup) [inline cva6/sim/cva6.yaml:57]: I call these "magic numbers", but they work without explanation.  We should either not use them, or use variable names that have some meaning.  Also, it is likely this number must match another number in a separate file.   It would be useful if this file and the others that need this magic number to obtain its value from a common source.

**Review by MikeOpenHWGroup — COMMENTED** [review]
Looks good @JeanRochCoulon.  I think that @ASintzoff should review and approve this PR before I merge it.  Also, I have one question about `cva6/sim/cva6.yaml`.

File: cva6/sim/cva6.py
Comment (ASintzoff) [inline cva6/sim/cva6.py:871]: Please use canonical order for `args.isa` (`rv32imafc`) to be sure compilers will recognize correctly the extensions.

File: cva6/sim/cva6.yaml
Comment (JeanRochCoulon) [reply cva6/sim/cva6.yaml]: @MikeOpenHWGroup , This modification has been removed from the current PR. New #1039 PR contains it, with the explanation of "magic numbers"

File: cva6/sim/cva6.py
Comment (JeanRochCoulon) [reply cva6/sim/cva6.py]: Corrected. Thanks

**Review by ASintzoff — APPROVED** [review]

**Pull request merged**
